source/production/F0.Minesweeper.Components/Logic/Cell/CellStatusTransition.cs

```diff
@@ -0,0 +1,33 @@
+using F0.Minesweeper.Components.Abstractions.Enums;
+
+namespace F0.Minesweeper.Components.Logic.Cell
+{
+	internal class CellStatusTransition
+	{
+		private readonly CellStatusType processState;
+		private readonly MouseButtonType command;
+
+		public CellStatusTransition(CellStatusType processState, MouseButtonType command) 
+		{
+			this.processState = processState;
+			this.command = command;
+		}
+
+		public override int GetHashCode()
+		{
+			unchecked
+			{
+				return 23 + 19 * processState.GetHashCode() + 19 * processState.GetHashCode();
```

Comment:
Nice to know this exists. But actually even nicer to know: due to this comment I noticed that equals used `processState` twice instead of once `processState` and once `command`. Changed both things